core/tee/tee_fs_rpc.c

```diff
@@ -27,12 +27,15 @@
 
 #include <assert.h>
 #include <kernel/thread.h>
+#include <kernel/tee_misc.h>
```

Comment:
major: shift <kernel/tee_misc.h> above <kernel/thread.h>
and shift <tee/tee_svc_storage.h> below <tee/tee_pobj.h> at line 37.